resources/function_help/json/LIKE

```diff
@@ -3,14 +3,19 @@
   "type": "operator",
   "description": "Returns 1 if the first parameter matches the supplied pattern. Works with numbers also.",
   "arguments": [
-	{"arg":"string/number","description":"value"},
-	{"arg":"pattern","description":"pattern to compare value with"}
+    {"arg":"string/number","description":"value"},
+    {"arg":"pattern","description":"pattern to compare value with"}
```

Comment:
One quick thing I noticed - it'd be good to mention the escaping in the description for the pattern argument